### ISSUE TITLE
 gcp-iam-audit-log-v1 Policy Does Not Work in Forseti

### ISSUE THREAD

**Issue by blueandgold** [forseti-security/forseti-security#3208]
Opening this in Forseti repo, since this affects Forseti and not other Config Validator clients.

The problem is that [gcp-iam-audit-log-v1 policy](https://github.com/forseti-security/policy-library/blob/master/policies/templates/gcp_iam_audit_log.yaml) does not work in forseti because the `audit_config` field [gets ignored](https://github.com/forseti-security/forseti-security/blob/dev/google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py#L94) when converting from CAI json to proto.

**Comment by blueandgold** [comment]
@joecheuk is there a reason why we do the ignoring of fields here, while the other CV clients does not?

@katze120 for FYI.

**Comment by katze120** [comment]
In terraform validator and cft scorecard, [the conversion leverages terraform validator converter](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/cli/scorecard/violations.go#L94-L95) which [only takes bindings field from iam_policy](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/cli/vendor/github.com/GoogleCloudPlatform/terraform-validator/converters/google/convert.go#L226-L235) and implicitly dropping audit_configs field.
FYI @morgante 

Is there any CV client that works fine with audit_configs field?

**Comment by morgante** [comment]
@katze120 Are you sure scorecard doesn't provide it? Scorecard uses the type from tfvalidator but nothing else.

**Comment by katze120** [comment]
yes, from actual test results and also the [Asset struct definition](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/cli/vendor/github.com/GoogleCloudPlatform/terraform-validator/converters/google/convert.go#L51-L86) in addition to links provided above

When I bypass tfvalidator and directly try to convert `interface{}` variable to proto, it gives me `Error: validating: FCV: converting asset  to proto: unmarshaling to proto: unknown field "audit_configs" in iam.Policy`

**Comment by morgante** [comment]
Got it, so we're likely encountering the same issue as Forseti (the protodef for IAM policies doesn't include audit_configs).
